client/src/components/new-request/analyze-results.vue

```diff
@@ -735,27 +751,46 @@ export default class AnalyzeResults extends Vue {
 
 </script>
 
-<style scoped lang="sass">
-#name-search-bar
-  margin: -20px 0 20px 0
-  max-height: 40px
-  min-height: 40px
-.action
-  color: $error !important
-.approved
-  color: $approved !important
-.error-message
-  color: red
-  margin-left: 20px
-  margin-top: 15px
-.modal-activator
-  background-color: unset !important
-  color: $link !important
-  cursor: pointer !important
-  letter-spacing: unset !important
-  text-decoration: underline
-  text-transform: none !important
-.strike
-  text-decoration-line: line-through
+<style scoped lang="scss">
+@import '@/assets/scss/theme.scss';
 
+#analyze-results-container {
+  max-width: 1140px;
+}
+#name-search-bar {
+  margin: -20px 0 20px 0;
+  max-height: 40px;
+  min-height: 40px;
+}
+.action {
+  color: red !important;
```

Comment:
I think we should centralize colours and use variables. Thoughts?

Reply:
for sure, as in add this to our Scss theme and apply the colour variable here?

Reply:
Updated!

Reply:
In this case, I know we already have 2 reds defined. Is this a necessary 3rd one?

Reply:
o shoot, i didn't notice. I saw a `$caution` but didn't port the `$error` over to the scss
Will fix

Reply:
ah we also have duplication classes. Two class names but same colour. Once again, not a downside i suppose if we need to differentiate between an error colour and action colour.. but currently they are the same.

Reply:
I agree that 2 names for the same colour is OK if we need to differentiate them in use.

Reply:
So we still have 2 red (and I'm added a third) :(
1. $caution is red `#bf4803`
2. $BCgovInputError is red `#ff5252`
3. $secondary (used for red/white buttons) is `d3272c` (per Tracey's spec)

We should try to get this down to only 1 or 2... in a future commit.

Reply:
PS I found that number 3 is plenty dark enough for warning icons and outline buttons.

Reply:
Definitely!